pyfar/classes/audio.py

```diff
@@ -732,12 +776,22 @@ def times(self):
     @property
     def frequencies(self):
         """Frequencies of the discrete signal spectrum."""
-        return np.atleast_1d(fft.rfftfreq(self.n_samples, self.sampling_rate))
+        if self._complex:  # assume the time domain data were complex-valued
+            # such that we need a two-sided Fourier spectrum
```

Comment:
```suggestion
        if self._complex:
            # assume the time domain data were complex-valued
            # such that we need a two-sided Fourier spectrum
```

Reply:
See above. I would suggest to use `self.complex`